src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/test/Configuration/ServiceCollectionExtensionTests.cs

```diff
@@ -35,5 +38,61 @@ public void ClientProxyTest_With_Missing_ProxyURL_ThrowsException()
 
         // Act & Assert
         Assert.Throws<ConfigurationException>(() => serviceCollection.AddElectricityMapsForecastDataSource(configuration.DataSources()));
+        Assert.Throws<ConfigurationException>(() => serviceCollection.AddElectricityMapsEmissionsDataSource(configuration.DataSources()));
+    }
+
+    [Test]
+    public void ClientProxyTest_With_No_ProxyURL_AddsServices()
+    {
+        // Arrange
+        var settings = new Dictionary<string, string> {
+            { ForecastDataSourceKey, ForecastDataSourceValue },
+            { EmissionsDataSourceKey, EmissionsDataSourceValue },
+            { HeaderKey, AuthHeader },
+            { TokenKey, DefaultTokenValue },
+            { UseProxyKey, "false" },
+        };
+        var configuration = new ConfigurationBuilder()
+            .AddInMemoryCollection(settings)
+            .AddEnvironmentVariables()
+            .Build();
+        var serviceCollection = new ServiceCollection();
+
+        // Act
+        var forecastService = serviceCollection.AddElectricityMapsForecastDataSource(configuration.DataSources());
+        var emissionsService = serviceCollection.AddElectricityMapsEmissionsDataSource(configuration.DataSources());
+
+        // Assert
+        Assert.NotNull(forecastService);
```

Comment:
nit: change for `Assert.That(...)` approach
```suggestion
 Assert.That(forecastService, Is.Not.Null);
```